knative-operator/deploy/resources/knativekafka/kafkasource-v0.17.1.yaml

```diff
@@ -337,7 +334,7 @@ spec:
         - name: CONFIG_LEADERELECTION_NAME
           value: config-leader-election-kafka
         - name: KAFKA_RA_IMAGE
-          value: gcr.io/knative-releases/knative.dev/eventing-contrib/kafka/source/cmd/receive_adapter@sha256:2bb36e62f1b4bd33eda26257371bc4f19db38da2062ebc712fce0fc42151f3ea
+          value: registry.svc.ci.openshift.org/openshift/knative-v0.17.1:knative-eventing-sources-kafka-source-adapter
```

Comment:
Ditto.

Reply:
hrm, yeah. ok;

we have vendored that file ourself - coming from the `openshift/knative-eventing-contrig` fork.

let me put the upstream images back